README.org

```diff
@@ -27,6 +27,8 @@ the 28.0 development tree.  See issue [[https://github.com/sharplispers/log4cl/i
 
 ** Enabling Log4Slime
 
+To enable Log4Sly run the following in your Lisp:
+
 #+BEGIN_SRC common-lisp
 (ql:quickload :log4slime)
```

Comment:
Shouldn't this be `log4cl.log4slime` given the previous changes?

Reply:
Yes, was relying on package nickname.  I have removed the nickname from both `log4cl.log4slime` and `log4cl.log4sly`, with a minor modification to the elisp code to spell out the package name.